Bottom: 8814da87e879054d9922c828b112b674b1d9d636
Top:    1b9f9a8931554329ad6d4a28128ee20d15b6e3fa
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-21 11:49:50 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5f..b14d49c42 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
